src/molecules/formfields/ToggleField/Readme.md

```diff
@@ -2,7 +2,14 @@
   <ToggleField
     htmlFor=''
     label='Toggle Choices'
-    toggleChoices={[ 'Choice 1', 'Choice 2' ]}
+    toggleChoices={[
+      {label: 'Choice 1', value: 1},
+      {label: 'Choice 2', value: 2}
+    ]}
     tooltipMessage='You can choose either of these awesome options!'
+    input={{
+      value: state.value,
+      onChange: (value) => setState({value})
```

Comment:
https://github.com/styleguidist/react-styleguidist/blob/master/docs/Documenting.md under 'Writing Code Examples' - I didn't know this either until last night, could be a nicer way of writing stateful examples rather than having to write a dummy component wrapper